modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
Not necessarily. To quote Mozilla

> This attribute can be used with any typical HTML element; it is not limited to elements that have an ARIA role assigned.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute